pygmt/tests/test_datasets_tutorial.py

```diff
@@ -58,3 +59,11 @@ def test_usgs_quakes():
     """
     data = load_usgs_quakes()
     assert data.shape == (1197, 22)
+
+
+def test_fractures_compilation():
+    """
+    Check that the dataset loads without errors.
+    """
+    data = load_fractures_compilation()
+    assert data.shape == (360, 2)
```

Comment:
Would be nice to add a few more assertion tests like so:

```suggestion
    assert data.shape == (360, 2)
    summary = data.describe()
    assert summary.loc["min", "length"] == 98.6561
    assert summary.loc["max", "length"] == 984.652
    assert summary.loc["min", "direction"] == 0.0
    assert summary.loc["max", "direction"] == 360.0

```

Reply:
Thanks for the suggestions @weiji14.